tools/bin/build_report.py

```diff
@@ -3,8 +3,13 @@
 #
 
 """
+Report Connector Build Status to Slack
+
 All invocations of this script must be run from the Airbyte repository root.
 
+BEFORE RUNNING THIS SCRIPT:
+- Ensure you have read the documentation on how this system works: https://internal-docs.airbyte.io/Generated-Reports/Build-Status-Reports
```

Comment:
👍 I think it's fine to reference internal docs in the OSS repo.  This part of the codebase is tooling.